core/imds/src/main/java/software/amazon/awssdk/imds/Ec2MetadataResponse.java

```diff
@@ -30,26 +30,26 @@
 /**
  * The class is used for response handling and parsing the metadata fetched by the get call in the {@link Ec2MetadataClient}
  * interface.
- * The class provides convenience methods to the users to parse the metadata as a String, List and Document.
+ * The class provides convenience methods to the users to parse the metadata as a String, List and Document (json).
```

Comment:
What do we mean by "Document (json)"?

Reply:
what I mean is that the method `asDocument` returns a `Document` interface, which is a representation of json a format.

Reply:
I'm not sure if Document is tied to JSON. Should we just link to the Document class? `{@link Document}`. Users can read more in the Document class

Reply:
Maybe I wasn't clear, the implementation of `asDocument` assumes the response is in json format, and returns a Document instance which is the representation of that that json. If it's not in json format, an exception will be thrown

Reply:
would the following be more clear?

>  The class provides convenience methods to the users to parse the metadata as a String and List. Also provides ways to parse a the metadata as Document type if it is in the json format.